docs/installation/data-preparation-scripts.rst

```diff
@@ -42,11 +42,11 @@ Download the USGS Collection 1 landsat scenes from any of the links below:
 
 The prepare script for collection 1 - level 1 data is available in
 `ls_usgs_prepare.py
-<https://github.com/opendatacube/datacube-dataset-config/blob/master/old-prep-scripts/ls_usgs_prepare.py>`_.
+<https://github.com/opendatacube/datacube-dataset-config/blob/main/old-prep-scripts/ls_usgs_prepare.py>`_.
```

Comment:
Ideally, we should update all this for USGS Landsat Collection 2.  

If Collection 1 is still available, it's probably safe to assume it won't be for much longer - and we should at least state this here.

Reply:
Might be better to migrate some of the product-specific docs from over here to replace this: https://github.com/opendatacube/datacube-dataset-config

I don't know that any of the prep scripts have been used for like ... 5+ years!

Reply:
Is it worth keeping this page at all, beyond linking to the datacube-dataset-config repo?

Reply:
I don't think so! The scripts probably don't work...